app/views/layouts/hera/_sidebar_header.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="sidebar-header">
+  <%= link_to main_app.root_path, class: 'd-flex' do %>
```

Comment:
since this will only be accessed in-project do we need `main_app`

Reply:
We're not using this file anymore. I'm not sure why/how it ended up in this branch when it's not in the base branch. Checking with Matt.

Reply:
Looks like a bad merge, it's removed now.